analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -224,6 +215,31 @@ def add_arguments_to_parser(parser):
                              "values are: {0}".format(
                                  ', '.join(REVIEW_STATUS_VALUES)))
 
+    group = parser.add_argument_group("file filter arguments")
+    group = group.add_mutually_exclusive_group()
+
+    group.add_argument('-i', '--ignore', '--skip',
+                       dest="skipfile",
+                       required=False,
+                       default=argparse.SUPPRESS,
+                       help="Path to the Skipfile dictating which project "
+                            "files should be omitted from analysis. Please "
+                            "consult the User guide on how a Skipfile "
+                            "should be laid out.")
+
+    group.add_argument('--file',
+                       nargs='*',
```

Comment:
```suggestion
                       nargs='+',
```
This could also be changed at `CodeChecker cmd diff`. `CodeChecker analyze` also has this `--file` flag and that requires at least one file too.

Reply:
I changed it only at the parse, but we can change it on the diff to but in a separate PR.